packages/commonwealth/client/scripts/views/pages/view_thread/linked_proposal.tsx

```diff
@@ -0,0 +1,31 @@
+import {
```

Comment:
for new created files and components, let's use PascalCase, so this should be `LinkedProposal.tsx`